core/src/main/scala/org/apache/spark/util/collection/ExternalSorter.scala

```diff
@@ -123,43 +134,28 @@ private[spark] class ExternalSorter[K, V, C](
   // grow internal data structures by growing + copying every time the number of objects doubles.
   private val serializerBatchSize = conf.getLong("spark.shuffle.spill.batchSize", 10000)
 
-  private def getPartition(key: K): Int = {
```

Comment:
This was just moved higher up in the file so that it was next to the the other partitioner-related variables.